backend/src/main/java/com/htecgroup/skynest/controller/UserController.java

```diff
@@ -352,4 +352,70 @@ public ResponseEntity<Boolean> changePassword(
     log.info("User with id {} successfully changed their password", uuid);
     return ResponseEntity.ok(true);
   }
+
+  @Operation(summary = "Disable user")
+  @ApiResponses(
+      value = {
+        @ApiResponse(
+            responseCode = "200",
+            description = "User successfully disabled",
+            content = {
+              @Content(
+                  mediaType = "application/json",
+                  schema = @Schema(implementation = String.class),
+                  examples = {@ExampleObject(value = "true")})
+            }),
+        @ApiResponse(
+            responseCode = "404",
+            description = "User not found",
+            content = {
+              @Content(
+                  mediaType = "application/json",
+                  schema = @Schema(implementation = ErrorMessage.class),
+                  examples = {
+                    @ExampleObject(
+                        value =
+                            "{\"messages\":[\"User with id a6fd6d95-0a60-43ff-961f-2b9b2ff72f95 doesn't exist\"],"
+                                + " \"status\": \"404\","
+                                + " \"timestamp\": \"2022-06-07 16:18:12\"}")
+                  })
+            }),
+        @ApiResponse(
+            responseCode = "403",
+            description = "User not verified",
+            content = {
+              @Content(
+                  mediaType = "application/json",
+                  schema = @Schema(implementation = ErrorMessage.class),
+                  examples = {
+                    @ExampleObject(
+                        value =
+                            "{\"messages\":[\"Not verified user can't be enabled.\"],"
+                                + " \"status\": \"403\","
+                                + " \"timestamp\": \"2022-06-07 16:18:12\"}")
+                  })
+            }),
+        @ApiResponse(
+            responseCode = "409",
+            description = "User already disabled",
+            content = {
+              @Content(
+                  mediaType = "application/json",
+                  schema = @Schema(implementation = ErrorMessage.class),
+                  examples = {
+                    @ExampleObject(
+                        value =
+                            "{\"messages\":[\"User is already disabled.\"],"
+                                + " \"status\": \"409\","
+                                + " \"timestamp\": \"2022-06-07 16:18:12\"}")
+                  })
+            })
+      })
+  @PreAuthorize("hasAuthority(T(com.htecgroup.skynest.model.entity.RoleEntity).ROLE_ADMIN)")
+  @PutMapping("/disable/{uuid}")
```

Comment:
will we always have operations before ids?

in other words, /users/disable/{uuid} instead of /users/{uuid}/disable

not sure which one makes more sense

Reply:
For me, this way is more natural

Reply:
/users/{uuid}/disable you should go with this, this is actually by REST

Reply:
Okay then, i'll make the change everywhere.

Reply:
maybe is better to be like this /users/{userId}/disable
this is much more readable

Reply:
So this was my point, after some resource it goes resource id, so you never gonna do something like this 
/users/function/{userId}/{functionId}, correct way is /users/{userId}/function/{functionId}